sos/plugins/sapnw.py

```diff
@@ -39,7 +39,8 @@ def collect_list_instances(self):
         # Cycle through all the instances, get 'sid', 'instance_number'
         # and 'vhost' to determine the proper profile
         for inst_line in inst_out['output'].splitlines():
-            if "DAA" not in inst_line:
+            if "DAA" not in inst_line and not
+                    inst_line.startswith("No instances found")):
```

Comment:
The trailing `)` is causing the LGTM failure. A leading `(` at the beginning of the conditional is probably the best soluition.

Reply:
Hi, thank you, I made the change.